index.html

```diff
@@ -40,57 +41,63 @@ <h1> through <h6>
 
 -->
 
-<!-- company name -->
-Sweet Eats Bakery
-
-<!-- Navigation -->
-About
-Cookies
-Weddings
-Catering
-Contact
-
-<!-- Marketing header -->
-About Sweet Eats Bakery
-
-Marzipan jelly-o macaroon I love macaroon jujubes. Candy candy canes jujubes I love ice cream croissant. Lollipop donut icing I love liquorice gummi bears marzipan. Dragée jelly beans apple pie cotton candy lemon drops pastry candy powder.
-
-<!-- content -->
-Cupcakes
-
-Candy apple pie lemon drops sweet roll danish. Tiramisu candy canes jelly-o jelly beans cotton candy pastry pudding tootsie roll carrot cake. Dessert sesame snaps brownie lemon drops cookie donut dragée. Toffee pie fruitcake powder gingerbread pudding.
+    <!-- company name -->
+    <h1>Sweet Eats Bakery</h1>
 
-Cookies
+    <!-- Navigation -->
+    <nav>
+        <a href="#">About</a>
+        <a href="#">Cookies</a>
+        <a href="#">Weddings</a>
+        <a href="#">Catering</a>
+        <a href="#">Contact</a>
+    </nav>
+    <!-- Marketing header -->
+    <h2>About Sweet Eats Bakery</h2>
 
-Cookie soufflé dessert carrot cake fruitcake halvah pudding cake. Marzipan topping bear claw soufflé gingerbread biscuit. Tiramisu gummi bears chocolate bar sugar plum icing muffin cake jelly-o icing.
+    <p>Marzipan jelly-o macaroon I love macaroon jujubes. Candy candy canes jujubes I love ice cream croissant. Lollipop donut icing I love liquorice gummi bears marzipan. Dragée jelly beans apple pie cotton candy lemon drops pastry candy powder.</p>
 
-Catering
+    <!-- content -->
+    <section>
+        <h2>Cupcakes</h2>
 
-Sweet jelly-o apple pie powder jelly beans. Pastry sweet roll cake jujubes halvah soufflé brownie. Toffee cookie lemon drops jelly beans chocolate bar. Wafer muffin jujubes danish tart danish chocolate wafer.
+        <p>Candy apple pie lemon drops sweet roll danish. Tiramisu candy canes jelly-o jelly beans cotton candy pastry pudding tootsie roll carrot cake. Dessert sesame snaps brownie lemon drops cookie donut dragée. Toffee pie fruitcake powder gingerbread
+            pudding.</p>
 
-Contact
+        <h2>Cookies</h2>
 
-yum@sweeteatsbakery.com
-202-555-0144
-4209 Bobcat Drive
-Madison, WI, 48219
+        <p>Cookie soufflé dessert carrot cake fruitcake halvah pudding cake. Marzipan topping bear claw soufflé gingerbread biscuit. Tiramisu gummi bears chocolate bar sugar plum icing muffin cake jelly-o icing.</p>
 
-<!-- footer -->
-Sweet Eats Bakery
+        <h2>Catering</h2>
 
-About
-Cookies
-Weddings
-Catering
-Contact
+        <p>Sweet jelly-o apple pie powder jelly beans. Pastry sweet roll cake jujubes halvah soufflé brownie. Toffee cookie lemon drops jelly beans chocolate bar. Wafer muffin jujubes danish tart danish chocolate wafer.</p>
+    </section>
+    <address>
+<h3>Contact</h3>
```

Comment:
If you use that prettify extension, it will handle indentation for you. So that these blocks of code will be easier to read.